VocaDb.ResXFileCodeGenerator.Tests/GeneratorTests.cs

```diff
@@ -108,7 +108,7 @@ namespace Resources
         public static string? CreateDateDescending => ResourceManager.GetString(nameof(CreateDateDescending), CultureInfo);
     }}
 }}";
-		using var resxStream = new MemoryStream(Encoding.UTF8.GetBytes(Text));
+		using var resxStream = new StringReader(Text);
```

Comment:
Can you please explain why you replaced `MemoryStream` with `StringReader`? I guess because of performance reasons?

Reply:
Because the built-in method of reading the file uses SourceText object, and there was no particular reason to convert to utf8 bytes just to convert it back a couple of lines later.